src/generic/components/ecl-navigation/ecl-navigation-side/ecl-navigation-side.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Side navigation related behaviors.
+ */
+
+import stickybits from 'stickybits';
+
+/**
+ * @param {object} options Object containing configuration overrides
+ */
+export const navigationSide = ({
+  stickySelector: stickySelector = '.ecl-navigation-side__toggle',
+} = {}) => {
+  // SUPPORTS
+  if (
+    !('querySelector' in document) ||
+    !('addEventListener' in window) ||
+    !document.documentElement.classList
+  )
+    return null;
+
+  // ACTIONS
+  function initSticky() {
+    // init sticky menu
+    // eslint-disable-next-line no-undef
+    stickybits(stickySelector, { useStickyClasses: true });
+  }
+
+  function scrollToTop() {
+    document
+      .getElementsByClassName(stickySelector.substring(1))[0]
```

Comment:
Please add an additional check the element exists before using it, as if the selector does not exist, it will crash.